rollup.config.js

```diff
@@ -34,18 +34,18 @@ export default [
     ],
   },
   // web worker
-  {
-    input: 'src/workers/parquetWorker.ts',
-    output: {
-      file: 'public/build/worker.min.js',
-      format: 'umd',
-      sourcemap: true,
-    },
-    plugins: [
-      commonjs(),
-      resolve({ browser: true }),
-      terser(),
-      typescript(),
-    ],
-  },
+  // {
+  //   input: 'src/workers/parquetWorker.ts',
+  //   output: {
+  //     file: 'public/build/worker.min.js',
+  //     format: 'umd',
+  //     sourcemap: true,
+  //   },
+  //   plugins: [
+  //     commonjs(),
+  //     resolve({ browser: true }),
+  //     terser(),
+  //     typescript(),
+  //   ],
+  // },
```

Comment:
Maybe a different PR, but if we're using vite for components, maybe should use vite instead of rollup for cli too? (remove rollup)

Reply:
added to the list of tasks. Maybe with https://www.npmjs.com/package/vite-node